src/main/scala/chisel3/iotesters/TesterOptions.scala

```diff
@@ -127,6 +129,11 @@ trait HasTesterOptions {
     .abbr("tts")
     .foreach { x => testerOptions = testerOptions.copy(testerSeed = x) }
     .text("provides a seed for random number generator")
+
+  parser.opt[Unit]("suppress-verilator-vcd")
+    .abbr("tsvv")
+    .foreach { _ => testerOptions = testerOptions.copy(suppressVerilatorVCD = true) }
+    .text(s"provides a way to turn of the default generation of vcd files in verilator simulations")
```

Comment:
should be `provides a way to turn off the default generation of vcd files in verilator simulations`